frontend/src/libs/helpers/get-formatted-time.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { format } from 'date-fns';
+
+const START_POINT = 0;
+
+const TIME_FORMATS = {
+  MM_SS: 'mm:ss',
+  HH_MM_SS: 'hh:mm:ss',
+};
+
+const getFormattedTime = (timeInSeconds: number): string => {
+  const date = new Date(START_POINT);
+  date.setHours(START_POINT);
+
+  date.setSeconds(timeInSeconds);
+  const hasHours = date.getHours();
+
+  return format(date, hasHours ? TIME_FORMATS.HH_MM_SS : TIME_FORMATS.MM_SS);
```

Comment:
The current format should be received from props and also put Date as argument

Reply:
Also, I have problems with timeZone, because when I tried to use "hh:mm:ss" format I saw +3 hours which is related to my local timezone. So, it seems to me that such lib isn't a good way to implement Duration via Date or, maybe, I do it in wrong way. So, please, help me with this.

Reply:
It should be something like here
<img width="624" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/43109766/d569235e-1d95-4af7-a3c3-100f3e5d3aeb">

Reply:
But I need only time format not day?  Am I wrong? Soorry, I got confused with this.

Reply:
It may show me something like: "1 Jan", but I need just time duration like "05:45"

Reply:
Ok, let me check

Reply:
So, please follow
```js
import { format } from "date-fns";

const MILLISECONDS_IN_SECOND = 1000;
const SECONDS_IN_MINUTE = 60;

const getFormattedDate = (date, formatType) => {
  return format(date, formatType);
};

const getTimeOffset = (seconds) => {
  const timeZoneOffsetInMinutes = new Date().getTimezoneOffset();

  const timeZoneOffsetInMS =
    timeZoneOffsetInMinutes * SECONDS_IN_MINUTE * MILLISECONDS_IN_SECOND;
  const offset = new Date(
    seconds * MILLISECONDS_IN_SECOND + timeZoneOffsetInMS
  );

  return getFormattedDate(offset, "HH:mm:ss");
};

console.log(getTimeOffset(110)); // 00:01:50 
```

Reply:
@v0ldemar01 I need one more clarification because it doesn't fix one problem. If we want to get formatType from props, and pass format "mm:ss" (without hours), but we have time in seconds that are bigger than hour we see this. (I pass 50000 seconds it must be 01:53:20, but got just 53:20). So in this case we have 2 options:
1. Do not pass format type by props ( and calculate if we have hours then the format will be "hh:mm:ss" otherwise just "mm:ss" 
2. We need to find another way to fix it.
So what should I do?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/a12ed58a-9f9f-45dd-af42-e1d1af21a871)

Reply:
Okey, let's it be the first one)